Composer/packages/adaptive-form/src/components/fields/OpenObjectField/ObjectItem.tsx

```diff
@@ -91,6 +91,7 @@ const ObjectItem: React.FC<ObjectItemProps> = ({
             {...rest}
             definitions={definitions}
             id={`${name}.value`}
+            label={false}
```

Comment:
Does this effect other object items? I think it should be based on if the fields are inline or not.

Reply:
**ObjectItem** is only used in **OpenObjectField** and this component already has headers (highlighted code in screenshot below):

![image](https://user-images.githubusercontent.com/66701106/97217041-e54b3d00-1783-11eb-8edd-d0c1ddab18b9.png)